charts/backbeat/values.yaml

```diff
@@ -31,6 +31,49 @@ api:
   tolerations: []
   affinity: {}
 
+lifecycle:
+  conductor:
+    replicaCount: 1
+    cronRule: "0 */5 * * * *"
+    resources: {}
+    nodeSelector: {}
+    tolerations: []
+    affinity: {}
+
+  consumer:
+    replicaCount: 1
+    group: backbeat-lifecycle-consumer-group
+    resources: {}
+    nodeSelector: {}
+    tolerations: []
+    affinity: {}
+
+  producer:
+    replicaCount: 1
+    group: backbeat-lifecycle-producer-group
+    resources: {}
+    nodeSelector: {}
+    tolerations: []
+    affinity: {}
+
+  zookeeper:
```

Comment:
On one hand, some want to use 'functional' names for objects etc. instead of service names. On the other hand, we use service names in configuration. This is... confusing. And also, hard to change later.